src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    {person.name && (
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+    )}
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried && (
+        <>
+          {person.sex === 'm'
+            ? `${person.partnerName} is my wife`
+            : `${person.partnerName} is my husband`
```

Comment:
Let's create a separate variable `partner` and use it here to make JSX cleaner

`const partner = person.sex === 'm' ? 'wife' : 'husband'`